.github/workflows/coding-standards.yml

```diff
@@ -9,8 +9,13 @@ on:
     branches:
       - "*.x"
 
+permissions:
+  contents: read
+
 jobs:
   coding-standards:
+    permissions:
+      contents: none
```

Comment:
can we really check the coding standards without having access to the contents ?